aldryn_newsblog/admin.py

```diff
@@ -153,6 +153,7 @@ def add_view(self, request, *args, **kwargs):
 
 class NewsBlogConfigAdmin(
     AllTranslationsMixin,
+    VersionedPlaceholderAdminMixin,
     PlaceholderAdminMixin,
```

Comment:
`PlaceholderAdminMixin` is already part of `VersionedPlaceholderAdminMixin`, so, we don't need both.